ntbs-service/DataMigration/NotificationMapper.cs

```diff
@@ -372,17 +372,10 @@ private ContactTracing ExtractContactTracingDetails(dynamic rawNotification)
         private PreviousTbHistory ExtractPreviousTbHistory(dynamic rawNotification)
         {
             var details = new PreviousTbHistory();
```

Comment:
can we initilise it as 
`new PreviousTbHistory { 
  PreviouslyHadTb = Converter.GetStatusFromString(rawNotification.PreviouslyHadTb)
  ....
}`
?

Reply:
We could, but I've been avoiding this notation in this file so that when there's a runtime error we get the exact line which went wrong (e.g. the one setting `PreviouslyHadTb`). Using the notation you suggest, the exception is thrown for the whole constructor, so it could be any of the property setters that go wrong and we couldn't tell which one from the stack trace alone